pandas/core/frame.py

```diff
@@ -3127,9 +3127,8 @@ def sort_values(self, by, axis=0, ascending=True, inplace=False,
                     kind='quicksort', na_position='last'):
 
```

Comment:
can you update the ``.sort_values`` doc-string to look like others, e.g. look at ``sortlevel`` (for the axis arg)